### PR TITLE
multiple pages(asciinema to atoum) : Add Korean Translation

### PR DESCRIPTION
**Pull request by Proscream**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by schneiderl** [comment]
Hi,
For those reviewing: Please be aware that some of the pages added here are also being added on #3555. Since this PR adds all the pages from #3555 (and more), I believe we can use this one for the reviewing process.

**Comment by Proscream** [comment]
@schneiderl sorry. i've seen it now. its corrected.

**Comment by Proscream** [comment]
I saw comment of # 3555. ToT

**Comment by Proscream** [comment]
I will commit to a new PR.

**Comment by einverne** [comment]
@Proscream Is this PR different from other? Which one should we take care?

**Comment by Proscream** [comment]
@einverne yes. Before I delete some pages, some pages are duplicated. So I deleted that pages, and schneiderl closed a PR #3555. Now there isnt duplicated page.

**Comment by Proscream** [comment]
@einverne Then I'll write in title that I translated A to B. That's okay?

**Comment by sbrl** [comment]
If you're going alphabetically, that would be great @Proscream :smiley_cat:

File: pages.ko/common/asciinema.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`asciinema auth`
+
+- 새로운 녹음을 작성 (한 번 완료되면 사용자는 업로드하거나 로컬로 저장하라는 메시지가 표시됩니다):
+
+`asciinema rec`
+
+- 새 녹음을 만들고 로컬 파일에 저장:
+
+`asciinema rec {{path/to/file}}.cast`
```
Comment (sbrl) [inline pages.ko/common/asciinema.md]: Perhaps `path/to/file` here could be translated too? `cast_id`, `name`, `version` etc. in the `{{token syntax}}` too for example below. Basically anything in the `{{token_syntax}}` that doesn't affect the execution of the command can be translated if possible.

Comment (Proscream) [reply pages.ko/common/asciinema.md]: Of course you can translate path/to/file. However, in Korea, most folder or file names are in English. But I'll translate it if you want.

**Comment by schneiderl** [comment]
Hi @Proscream 
Were those last commits on purpose?

**Comment by Proscream** [comment]
@schneiderl yes. In #3573, we missed trailing lines, so we modfiy that.

File: pages.ko/common/asciinema.md (hunk 1 above)
Comment (sbrl) [reply pages.ko/common/asciinema.md]: Ah, I see. In other languages, we prefer to translate them. However, if in Korea you usually have file and folder names in English, I guess we should stick with the cultural norms.

Not speaking Korean means that I can only suggest things that might help :P

Comment (Proscream) [reply pages.ko/common/asciinema.md]: Korean programmers prefer to use English as it is for commands or terminology in computers (eg file names). But if there is no problem, I will translate it. Thank you for your comment!

File: pages.ko/common/asciinema.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# asciinema
+
+>  터미널 세션을 녹음하고 재생하며 선택적으로 asciinema.org에서 공유합니다.
```
Comment (einverne) [inline pages.ko/common/asciinema.md]: Just one space here.

```suggestion
> 터미널 세션을 녹음하고 재생하며 선택적으로 asciinema.org에서 공유합니다. 
```

File: pages.ko/common/asdf.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# asdf
+
+>  여러 패키지 버전을 관리하기 위한 CLI(Command Line Interface) 입니다.
```
Comment (einverne) [inline pages.ko/common/asdf.md]: same as above.

File: pages.ko/common/astronomer.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# astronomer
+
+>  Github 프로젝트의 봇 계정에서 불법적인 star를 탐지하는 도구.
```
Comment (einverne) [inline pages.ko/common/astronomer.md]: Same.

File: pages.ko/common/astyle.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# astyle
+
+> C, C ++, C # 및 Java 프로그래밍 언어에 대한 소스 코드 인덴터, 포맷터 및 미화기. 
+> 실행 시 원본 파일의 사본은 원래 파일 이름에 ".orig"가 추가된 상태로 작성된다. 
+> 더 많은 정보: <http://astyle.sourceforge.net/>.
+
+-  들여쓰기 당 4개의 공백의 기본 스타일을 적용하고 형식 변경 없도록 적용:
```
Comment (einverne) [inline pages.ko/common/astyle.md]:
```suggestion
- 들여쓰기 당 4개의 공백의 기본 스타일을 적용하고 형식 변경 없도록 적용:
```

**Comment by sbrl** [comment]
Are we read to merge @Proscream?

**Comment by Proscream** [comment]
@sbrl If you're asking me what I've read about merge, I've read it.
 In addition, I can't do translation work for a while because it's a university exam period. ToT
I'll try again soon.

**Comment by Proscream** [comment]
@sbrl  oh, if you want to say 'ready to', LGTM!

**Comment by sbrl** [comment]
Yeah, I was asking if we could merge this @Proscream :-)

Good luck with your exams!

**Pull request merged**